yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -33,8 +33,8 @@ func init() {
 type tableMigStatusOutputRow struct {
 	tableName          string
 	status             string
-	totalRowCount      int64
-	importedRowCount   int64
+	totalRowCount      float64
+	importedRowCount   float64
```

Comment:
Keep it `int64`.
Rename these two vars as `totalCount` and `importedCount`. Add a comment stating, the vars hold "row count" for `import data` and "byte count" for `import data file`.

Reply:
Done